pkg/secrets/secrets.go

```diff
@@ -80,3 +85,69 @@ func GenerateUserData(publicKey ssh.PublicKey) (*core.Secret, error) {
 
 	return userDataSecret, nil
 }
+
+// Encrypt creates an ecnrypted block of text from a plaintext message using the given key
+func Encrypt(plainText string, key []byte) (string, error) {
+	if key == nil {
+		return "", errors.New("encryption passphrase cannot be nil")
+	}
+
+	// prepare PGP block with a wrapper tag around the encrypted data
+	tag := "ENCRYPTED DATA"
+	msgBuffer := bytes.NewBuffer(nil)
+	encoder, _ := armor.Encode(msgBuffer, tag, nil)
+
+	writer, _ := openpgp.SymmetricallyEncrypt(encoder, key, nil, nil)
```

Comment:
can there be an error returned here?

Reply:
Yes an error can be returned and it needs to be handled